homeassistant/components/fritz/switch.py

```diff
@@ -46,9 +46,7 @@ async def _async_deflection_entities_list(
 
     _LOGGER.debug("Setting up %s switches", SWITCH_TYPE_DEFLECTION)
 
-    if (
-        call_deflections := avm_wrapper.data.get("call_deflections")
-    ) is None or not isinstance(call_deflections, dict):
```

Comment:
`avm_wrapper.data["call_deflections"]` is always initialized as empty dict `{}` so we do not need to check if the key exists, nor if it is a dict